ext/standard/proc_open.c

```diff
@@ -645,7 +700,7 @@ static int set_proc_descriptor_to_file(struct php_proc_open_descriptor_item *des
 	_close((int)fd);
 
 	/* simulate the append mode by fseeking to the end of the file
-	this introduces a potential race-condition, but it is the best we can do, though */
+	this introduces a potential race condition, but it is the best we can do */
 	if (strchr(ZSTR_VAL(zmode), 'a')) {
```

Comment:
To be able to build on Windows:
```suggestion
	if (strchr(ZSTR_VAL(file_mode), 'a')) {
```

Reply:
Thanks, updated.